tests/test_server.py

```diff
@@ -210,6 +210,8 @@ async def test_create_too_many_versions(client, server, n_versions_to_keep, n_ve
     result = await client.set_setting(tid=env_1_id, id=data.AVAILABLE_VERSIONS_TO_KEEP, value=n_versions_to_keep)
     assert result.code == 200
 
+    await client.create_environment(project_id=project_id, name="env_2")
```

Comment:
This doesn't add any value to the test case if it doesn't contain any resources, right?

Reply:
it does, this is how I reproduced it.

Reply:
then add a comment!